deps/rabbit/src/rabbit_table.erl

```diff
@@ -392,13 +392,16 @@ definitions() ->
 maybe_ensure_index_route_table() ->
```

Comment:
Unrelated to this patch, but I have a hard time making sens of the name "maybe ensure". Am I sure in the end that the table exists? Maybe, I don't know :-)